lib/codecs/src/decoding/format/gelf.rs

```diff
@@ -0,0 +1,203 @@
+use bytes::Bytes;
+use chrono::{DateTime, NaiveDateTime, Utc};
+use regex::Regex;
+use serde::{Deserialize, Serialize};
+use smallvec::{smallvec, SmallVec};
+use std::collections::HashMap;
+use value::Kind;
+use vector_core::{
+    config::{log_schema, DataType},
+    event::Event,
+    schema,
+};
+
+use super::Deserializer;
+
+/// GELF Message fields. Definitions from https://docs.graylog.org/docs/gelf
+#[rustfmt::skip]
```

Comment:
Why are we not formatting this struct?

Reply:
Just a personal preference thing, to see the variable names, types and values aligned column-wise.
I can remove it if its not adhering to code standards of vector etc.

Reply:
Yeah, not particularly bothered and keeping things aligned is nice - but I think we do just do `rustfmt` regardless. A quick grep only shows that skip in the VRL parser